### PR TITLE
Tika 3272 - Remove usage of rotation.py and Python dependency

### PR DESCRIPTION
**Pull request by peterkronenberg**
Remove usage of rotation.py and Python dependency.  Allow **ApplyRotation** and **EnableImageProcessing** to be independent

### CODE REVIEW

File: tika-parsers/tika-parsers-classic/tika-parsers-classic-modules/tika-parser-ocr-module/src/main/java/org/apache/tika/parser/ocr/TesseractOCRParser.java [hunk 1]
```diff
@@ -43,7 +43,7 @@
 import org.xml.sax.helpers.DefaultHandler;
 
 import javax.imageio.ImageIO;
-import java.awt.Image;
+import java.awt.*;
```
Comment (tballison) [inline tika-parsers/tika-parsers-classic/tika-parsers-classic-modules/tika-parser-ocr-module/src/main/java/org/apache/tika/parser/ocr/TesseractOCRParser.java:46]: Please avoid wildcard imports

File: tika-parsers/tika-parsers-classic/tika-parsers-classic-modules/tika-parser-ocr-module/src/main/java/org/apache/tika/parser/ocr/tess4j/ImageDeskew.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
+package org.apache.tika.parser.ocr.tess4j;
```
Comment (tballison) [inline tika-parsers/tika-parsers-classic/tika-parsers-classic-modules/tika-parser-ocr-module/src/main/java/org/apache/tika/parser/ocr/tess4j/ImageDeskew.java:1]: Please add Apache License header.

Also, please add a comment along the lines of "copied and pasted from" and then a link to the tess4j code

File: tika-parsers/tika-parsers-classic/tika-parsers-classic-modules/tika-parser-ocr-module/src/main/java/org/apache/tika/parser/ocr/tess4j/ImageUtil.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
+package org.apache.tika.parser.ocr.tess4j;
```
Comment (tballison) [inline tika-parsers/tika-parsers-classic/tika-parsers-classic-modules/tika-parser-ocr-module/src/main/java/org/apache/tika/parser/ocr/tess4j/ImageUtil.java:1]: Ditto...license header and comment "copied from..."

File: tika-parsers/tika-parsers-classic/tika-parsers-classic-modules/tika-parser-ocr-module/src/main/java/org/apache/tika/parser/ocr/tess4j/ImageUtil.java [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+
+import org.apache.tika.parser.ocr.TesseractOCRParser;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.awt.*;
```
Comment (tballison) [inline tika-parsers/tika-parsers-classic/tika-parsers-classic-modules/tika-parser-ocr-module/src/main/java/org/apache/tika/parser/ocr/tess4j/ImageUtil.java:8]: please avoid wildcard imports

File: tika-parsers/tika-parsers-classic/tika-parsers-classic-modules/tika-parser-ocr-module/src/main/java/org/apache/tika/parser/ocr/TesseractOCRParser.java [hunk 5]
```diff
@@ -239,25 +239,20 @@ private void parse(TikaInputStream tikaInputStream, File tmpOCROutputFile,
             if (size >= config.getMinFileSizeToOcr() && size <= config.getMaxFileSizeToOcr()) {
 
             	// Process image
-            	if (config.isEnableImageProcessing()) {
+            	if (config.isEnableImageProcessing() || config.isApplyRotation()) {
```
Comment (tballison) [inline tika-parsers/tika-parsers-classic/tika-parsers-classic-modules/tika-parser-ocr-module/src/main/java/org/apache/tika/parser/ocr/TesseractOCRParser.java:242]: I think this makes more sense than the consistency check.  If the user selected either, go for it.

**Review by tballison — CHANGES_REQUESTED** [review]

**Comment by peterkronenberg** [comment]
New commit with requested changes (trying to figure out how to refresh the pull request)

**Pull request merged**